### PR TITLE
Add pgp import/generate choice screen DESKTOP-1607

### PR DESCRIPTION
**Pull request by awendland**
### What is this?

This adds a dumb component for the PGP import key/generate new key screen. Yay :tada:

Now also adds dumb component called `PgpIcon` which will provide these badged PGP key icons. Hoopla 🎊 

### What is up w/ the non master base?

This makes heavy use of some features in another open PR, so the diff is miniscule if based on that PR.

~~### To Do and Greater Problems
This is currently using the generic PGP icon for the options, instead of the compound PGP key + import/add icons which I am hoping design can fashion because these compound icons have non-uniform positioning (when including the pgp key + unfinished/placeholder overlay) which makes the effort to code them in a scalable manner seem in excess when compared to just using some SVG/PNG assets. These icons are also needed for other PGP PRs, **so it may be useful for all related tasks to just use the placeholder PGP key and then the integration PR can update them all in one fell swoop.** *Thoughts? @MarcoPolo @cjb (people who I think are working on PGP related tasks).*~~

@keybase/react-hackers

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 70415bb674d0...9bfe572a4f78 introduced some visual changes on linux:
 * Added: **New_Proof_Website-DNS_or_File**  
   New_Proof_Website-DNS_or_File rendered
 * Added: **New_Proof_PGP-Import_or_Generate**  
   New_Proof_PGP-Import_or_Generate rendered
 * Added: **Choice_List-Two_Choices**  
   Choice_List-Two_Choices rendered

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 70415bb674d0...27198ab0a335 introduced some visual changes on linux:
 * Added: **New_Proof_Website-DNS_or_File**  
   New_Proof_Website-DNS_or_File rendered
 * Added: **New_Proof_PGP-Import_or_Generate**  
   New_Proof_PGP-Import_or_Generate rendered
 * Added: **Choice_List-Two_Choices**  
   Choice_List-Two_Choices rendered

File: shared/profile/pgp/pgp-icon.desktop.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      right: 11,
+    },
+    type: 'iconfont-add',
+  },
+}
+
+const PgpIcon = ({type}: Props) => {
+  const overlayProps = supportedOverlays[type]
+  return (
+    <Box style={{display: 'flex', justifyContent: 'center', alignItems: 'center', position: 'relative', height: 48, width: 48}}>
```
Comment (chrisnojima) [inline shared/profile/pgp/pgp-icon.desktop.js]: globalMargins

File: shared/profile/pgp/pgp-icon.desktop.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      fontSize: 16,
+      color: globalColors.blue,
+      bottom: -1,
+      right: 11,
+    },
+    type: 'iconfont-add',
+  },
+}
+
+const PgpIcon = ({type}: Props) => {
```
Comment (chrisnojima) [inline shared/profile/pgp/pgp-icon.desktop.js]: I don't think we should have this class. we should just have design make a single icon for this and use Icon

Comment (awendland) [inline shared/profile/pgp/pgp-icon.desktop.js]: Sounds good. I'll ask some of those nice people over in design to throw something together.

Comment (awendland) [inline shared/profile/pgp/pgp-icon.desktop.js]: @chrisnojima Design is all OOO for the time being. Let's go with this for now and then we can change it out later when they get back if need be.

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 70415bb674d0...a16f094be40c introduced some visual changes on linux:
 * Added: **New_Proof_Website-DNS_or_File**  
   New_Proof_Website-DNS_or_File rendered
 * Added: **New_Proof_PGP-Import_or_Generate**  
   New_Proof_PGP-Import_or_Generate rendered
 * Added: **Choice_List-Two_Choices**  
   Choice_List-Two_Choices rendered

File: shared/profile/pgp/pgp-icon.desktop.js (hunk 2 above)
Comment (chrisnojima) [inline shared/profile/pgp/pgp-icon.desktop.js]: lets just choose a placeholder image so later it'll just be changing one string for another vs having to remove this component then

**Comment by chrisnojima** [comment]
👍

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 70415bb674d0...5ff3792039a5 introduced some visual changes on linux:
 * Added: **New_Proof_Website-DNS_or_File**  
   New_Proof_Website-DNS_or_File rendered
 * Added: **New_Proof_PGP-Import_or_Generate**  
   New_Proof_PGP-Import_or_Generate rendered
 * Added: **Choice_List-Two_Choices**  
   Choice_List-Two_Choices rendered

**Pull request merged**